src/JavaMustKnowStuff/Task2.java

```diff
@@ -0,0 +1,27 @@
+package JavaMustKnowStuff;
+
+import java.util.Scanner;
+
+public class Task2 {
+
+    static void ValidatePassword(String string){
+
+        String pattern = "^(?=.*[0-9])"
```

Comment:
was something said in password requirements about special symbols ? cause this patters as i see allows them, if not, all good

Reply:
2.	Write a Java method to validate whether a string is a valid password.
Note: Requirements of a valid password:
a.	The length must be at least 8 character and not exceed 15 characters;
b.	The password must consist of letters and digits;
c.	The password must have at least two uppercases and two digits.

Reply:
okay, in this case, because of point b.  The password must consist of letters and digits;
add please to pattern section that will exclude all special symbols